src/mol-script/transpilers/_spec/rasmol.spec.ts

```diff
@@ -0,0 +1,62 @@
+/**
```

Comment:
Is there anything in Rasmol expressions that is NOT supported in Jmol (actual Jmol, not the transpiler here which may miss some features)? I am wondering if it is necessary to have both or if Jmol is enough.

Reply:
I understand that Jmol supports all expressions RasMol does. As actual Jmol expression seems to be superset of RasMol,  I agree that they should be merged into one as Jmol if possible.

However, as for transpilers, I made RasMol transpiler support expression like [LYS]100:A.CA (<- C-alpha of 100th Lys on chain A ) and residue-range expression, which are left unsupported in Jmol transpiler.  The difference from actual RasMol expression is that residue name has to be in brackets and residue indexes has to be parenthesized. This is because I implemented these in similar manner as PyMOL selection macro using `[]`, `()`, `:` and `. `as keys to make them recognized as macro. In my implementation, RasMol is very similar to PyMOL rather than Jmol.

I tried to integrate these with Jmol expression, but I found it difficult for them to co-exist because Jmol and RasMol expression lacks property names  like _resi_ or _resn_ in PyMOL and _residue_ or _resname_ in VMD, and such raw expressions like _100_ or _LYS_ are confused with keywords.

Reply:
Update: 
Now it's made possible to use un-parenthesized resno-range in RasMol. This should be ported to Jmol.

Reply:
I found that my RasMol implementation cannot deal with negative-valued residue index because hyphen is used to represent the index range like 1-200. The problem is that Jmol spec test have an examination of negative-valued residue index, so it's impossible to simply import the implementation of residue range to Jmol from RasMol. 
There are three options:
(1) Import it and change spec of Jmol
(2) Give up importing
(3) Extend residue range selection to negatively valued index (desirable)

Reply:
Finally, I made it possible to perform residue range selection including negative-valued index in RasMol. This could be ported to Jmol.